projects/sandbox/src/app/home/home.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="home">
+  <div class="home-header">
+    <h1>Welcome to<br />Metis Sandbox!</h1>
+    <div class="circle-container-outer">
+      <ul class="circle-container">
+        <li class="orb-container"><a routerLink="/new" class="nav-orb labelled upload-orb"></a></li>
+        <li class="orb-container">
+          <a routerLink="/dataset" class="nav-orb labelled progress-orb"></a>
+        </li>
+        <li class="orb-container"><span class="nav-orb labelled is-active problem-orb"></span></li>
+        <li class="orb-container"><span class="nav-orb labelled is-active report-orb"></span></li>
+        <li class="orb-container"><span class="nav-orb labelled is-active problem-orb"></span></li>
+      </ul>
+    </div>
+  </div>
+  <div class="home-content">
+    <span>
+      The Sandbox allows you to:
+    </span>
+    <ol class="list">
+      <li>
+        test to see if your data will work within the Metis flow
```

Comment:
Personal taste, but could you start each sentence with a capital letter?
"Test to see..."
"Get insight into..."
"Preview your records..."

Reply:
I like proper capitalisation too, but technically those lines don't start sentences, so shouldn't be capitalised!